### ISSUE TITLE
Multiple `COUNT` queries conflict with each other

### ISSUE THREAD

**Issue by mhauru** [opendp/smartnoise-sdk#545]
I'm doing a query of the form
```
      SELECT
      COUNT(*) as num,
      COUNT(col_a) AS num_non_null_a,
      COUNT(col_b) AS num_non_null_b,
      COUNT(col_c) AS num_non_null_c,
      col_x,
      col_y
      FROM some_table
      WHERE blahblahblah
      GROUP BY col_x, col_y
```
and the four `COUNT` queries get messed up by I think the rewriter in such a way that all of them are actually doing `COUNT(*)` and thus report the same value. If I change the `COUNT(*)` to be the last of the four like so
```
      SELECT
      COUNT(col_a) AS num_non_null_a,
      COUNT(col_b) AS num_non_null_b,
      COUNT(col_c) AS num_non_null_c,
      COUNT(*) as num,
      col_x,
      col_y
      FROM some_table
      WHERE blahblahblah
      GROUP BY col_x, col_y
```
it works fine.

I suspect this might have something to do with something like conflicting names in `Rewriter`, but that's just a guess.

**Comment by joshua-oss** [comment]
Fix is pushed to PyPi smartnoise-sql==0.2.12

**Comment by joshua-oss** [comment]
Fixed
